tests/jballerina-unit-test/src/test/java/org/ballerinalang/test/expressions/invocations/FuncInvocationExprTest.java

```diff
@@ -146,6 +147,23 @@ public Object[][] invocationWithArgVarargMixTestFunctions() {
         };
     }
 
+    @Test(expectedExceptions = {BLangRuntimeException.class},
+            expectedExceptionsMessageRegExp = "error: Invalid.*")
+    public void testFuncWithNeverReturnTypeWithoutVariableAssignment1() {
+        BRunUtil.invoke(funcInvocationExpResult, "testFuncWithNeverReturnTypeWithoutVariableAssignment1");
+    }
+
+    @Test(expectedExceptions = {BLangRuntimeException.class},
+            expectedExceptionsMessageRegExp = "error: Invalid.*")
+    public void testFuncWithNeverReturnTypeWithoutVariableAssignment2() {
+        BRunUtil.invoke(funcInvocationExpResult, "testFuncWithNeverReturnTypeWithoutVariableAssignment2");
+    }
+
+    @Test(description = "Test function invocation without variable assignment")
```

Comment:
IMO, the description is not required. Or we need to improve it. The test name already provides more details in a way.